programs/lib/include/flexpret_swconfig.h

```diff
@@ -0,0 +1,8 @@
+#ifndef FLEXPRET_SWCONFIG_H
```

Comment:
This is a nice addition. Two comments:
1. Can we make it more user-configurable? Maybe this file is the default swconfig and we can let the user override the values. This could be done if we do
```
#if !defined(STACKSIZE_BITS)
#define STACKSIZE_BITS 11
#endif
```
Then the user could overwrite this config.

2. Could you add some tests with different STACK_SIZES which verifies that it works?

Reply:
1. Is done.
2. I'm not entirely sure how such a test would fit into the current test environment. Because I suppose I would have to run the same program multiple times with the STACKSIZE variable set to different value each time. Or did you have something else in mind?